### PR TITLE
Remove XML Namespaces

### PR DESCRIPTION
**Pull request by papeh**
Add a utility method to remove XML namespaces

---
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/libpalaso/1078)

### CODE REVIEW

File: SIL.Core/Xml/XmlUtils.cs [hunk 1]
```diff
@@ -1138,6 +1138,33 @@ private static bool IsLegalXmlChar(int codePoint)
...
+			RemoveNamespaces(element.DescendantsAndSelf());
+			return element;
+		}
+
+		private static void RemoveNamespaces(IEnumerable<XElement> elements)
+		{
+			foreach (var element in elements)
+			{
+				element.Attributes().Where(attribute => attribute.IsNamespaceDeclaration).ForEach(attribute => attribute.Remove());
+				element.Name = element.Name.LocalName;
```
Comment (ermshiperete) [inline SIL.Core/Xml/XmlUtils.cs:1164]: Why is this line necessary? Should we have a test for it?

**Review by ermshiperete — COMMENTED** [review]
This change should then be cherry-picked or merged into `master`

**Review by papeh — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/sillsdev/libpalaso/1078)* status: 0 of 2 files reviewed, 1 unresolved discussion (waiting on @ermshiperete)

___
*[SIL.Core/Xml/XmlUtils.cs, line 1164 at r1](https://reviewable.io/reviews/sillsdev/libpalaso/1078#-Ma5ZvEUYT1nwcPrqDvz-r1-1164:-MaAGO68Aa4Fee7r1g6T:b-e9z5f9) ([raw file](https://github.com/sillsdev/libpalaso/blob/5592aedd6ddcedc995585c5509f3014a9455702a/SIL.Core/Xml/XmlUtils.cs#L1164)):*
<details><summary><i>Previously, ermshiperete (Eberhard Beilharz) wrote…</i></summary><blockquote>

Why is this line necessary? Should we have a test for it?
</blockquote></details>

Because of the way XNode represents the XML in memory. The namespace is stored as an attribute as in the text, and also in the name of that element and all its descendants. Both of these lines are necessary for the current tests to pass, but you made me think of another helpful test.

**Review by ermshiperete — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/libpalaso/1078#-:-MaCXqaH7sMoRhw9wutr:bnfp4nl)* status: 1 of 2 files reviewed, all discussions resolved (waiting on @papeh)
